weblate/screenshots/tests.py

```diff
@@ -245,3 +248,43 @@ def test_translation_manipulations(self):
             {"source": source_pk},
         )
         self.assertEqual(screenshot.units.count(), 0)
+
+
+class ScreenshotVCSTest(RepoTestCase):
+    """Test class for syncing vcs screenshots in weblate."""
+
+    def setUp(self):
+        super().setUp()
+        self.component = self._create_component(
+            "json",
+            "intermediate/*.json",
+            screenshot_filemask="*.png",
+        )
+
+    def test_update_screenshots_from_repo(self):
+        repository = self.component.repository
+        last_revision = repository.last_revision
+
+        with open(TEST_SCREENSHOT, "rb") as handle, open(
+            os.path.join(repository.path, "test.png"), "wb"
+        ) as f:
+            f.write(handle.read())
```

Comment:
Better use Python library function for copying file:

```suggestion
        copyfile(TEST_SCREENSHOT, os.path.join(repository.path, "test.png"))
```